pkg/bff/admin.go

```diff
@@ -128,3 +132,226 @@ func (s *Server) Certificates(c *gin.Context) {
 
 	c.JSON(http.StatusOK, out)
 }
+
+const (
+	StartRegistration    = "Start the registration and verification process for your organization to receive an X.509 Identity Certificate and become a trusted member of the TRISA network."
+	CompleteRegistration = "Complete the registration process and verification process for your organization to receive an X.509 Identity Certificate and become a trusted member of the TRISA network."
+	SubmitTestnet        = "Review and submit your testnet registration."
+	SubmitMainnet        = "Review and submit your mainnet registration."
+	RenewCertificate     = "Your organization's %s X.509 Identity Certificate will expire on %s. Start the renewal process to receive a new X.509 Identity Certificate and remain a trusted member of the TRISA network."
+	CertificateRejected  = "Your organization's %s X.509 Identity Certificate has been rejected by TRISA. This means your organization is not a verified member of the TRISA network and cannot communicate with other members. Please contact TRISA at support@trisa.io for additional details and next steps."
+	CertificateRevoked   = "Your organization's %s X.509 Identity Certificate has been revoked by TRISA. This means your organization is no longer a verified member of the TRISA network and can no longer communicate with other members. Please contact TRISA at support@trisa.io for additional details and next steps."
+)
+
+// GetVASPs makes parallel calls to the admin APIs to retrieve VASP records from
+// testnet and mainnet. If testnet or mainnet are empty strings, this will simply
+// return a nil response for the corresponding network so the caller can distinguish
+// between a non registration and an error.
+func (s *Server) GetVASPs(ctx context.Context, testnetID, mainnetID string) (testnetVASP, mainnetVASP *pb.VASP, testnetErr, mainnetErr error) {
```

Comment:
True, we're relying on the admin endpoint here to retrieve the verification status and certificate info. I think this will fill the gap for right now but I agree it's not ideal and I think storing the attention state on the organization seems more maintainable.

Reply:
Could you create a spike story in Sprint 16 for us to figure out how we want to handle this better?